src/services/transactions_history.rs

```diff
@@ -107,7 +107,7 @@ fn fetch_backend_paged_txs(
 ) -> Result<Page<Transaction>> {
     let page_metadata = PageMetadata::from_url_string(page_url.as_ref().unwrap_or(&"".to_string()));
     let url = format!(
-        "{}/v1/safes/{}/all-transactions/?{}&queued=false",
+        "{}/v1/safes/{}/all-transactions/?{}&queued=false&executed=true",
```

Comment:
Do we need both? shouldn't `executed=true` be enough? 🤔 but should also not harm

Reply:
You are right. I think `queued=false` transactions should be a subset of `executed=true` but just to be extra safe 🤔 maybe instead of being safe it is just confusing :P I can change it if you prefer